src/Engine/DSP/ChannelFileWriter.h

```diff
@@ -65,7 +65,7 @@ class ENGINE_API ChannelFileWriter
 		bool WriteHeader(EFormat format, const Core::Array<Channel<double>*>& channels, FILE* file);
 		
 		// appends the last N sampels to the file
-		bool WriteSamples(EFormat format, const Core::Array<Channel<double>*>& channels, uint64 numSamples, FILE* file, const char* fileName);
+		bool WriteSamples(EFormat format, const Core::Array<Channel<double>*>& channels, uint64 numSamples, FILE* file, const int edfHandler = -1);
```

Comment:
agree. -1 is used since it's the case when file can not be opened, to check for valid/invalid handler. But since we always pass it having a default value does not make sense. 
Will adjust.